scripts/billing/sql/daily_compute_discount.sql

```diff
@@ -0,0 +1,15 @@
+/*
+Query returns compute cost including discounts on a daily basis
+*/
+
+SELECT
+  FORMAT_DATETIME("%Y%m%d", DATETIME(usage_end_time)) AS date_ymd, 
+  ROUND(SUM(cost - credits.amount) * 100) / 100 AS actual_cost
+FROM
+  `data-analytics-pocs.public.billing_dashboard_export`,
+  UNNEST(credits) AS credits
+WHERE service.id = "6F81-5844-456A"
```

Comment:
Can you add 2 spaces after the quotes and a single-line comment describing what this identifier is?

Example:
```sql
WHERE 
  service.id = '6F81-5844-456A'  -- Compute Engine
```